BanchoSharp/Multiplayer/MultiplayerLobby.cs

```diff
@@ -408,6 +373,195 @@ private void UpdateLobbyFromBanchoBotSettingsResponse(string banchoBotResponse)
 
 			OnPlayerSlotMove?.Invoke(new PlayerSlotMoveEventArgs(player, previousSlot, slotNum));
 		}
+		else if (banchoBotResponse.StartsWith("Players: "))
+		{
+			var playerCount = int.Parse(banchoBotResponse.Split("Players: ")[1]);
+
+			if (playerCount == 0)
+			{
+				// The "Players: <count>" should only come after a !mp settings request, so if we've gotten this, 
+				// and the count is 0, the "!mp settings" should have finished.
+
+				OnSettingsUpdated?.Invoke();
+			}
+			else
+			{
+				_playersRemainingCount = playerCount;
+			}
+		}
+		else if (banchoBotResponse.StartsWith("Slot "))
+		{
+			// Find the digit(s) within the first 8 characters
+			var slot = int.Parse((banchoBotResponse[..8].Where(c => char.IsDigit(c)).ToArray()));
+
+			// Find the first ' ' after the URL, since the URL is not padded with any spaces.
+			var playerNameBegin = banchoBotResponse.IndexOf(' ', banchoBotResponse.IndexOf("/u/")) + 1;
+
+			var name = banchoBotResponse.Substring(playerNameBegin, 16).TrimEnd();
```

Comment:
Please specify what object name is referring to in the variable name. e.g. `playerName`